src/js/CarGameManager.js

```diff
@@ -0,0 +1,107 @@
+import CarGameView from './CarGameView.js';
+
+import RacingCarGame from './Game/RacingCarGame.js';
+
+import { carNameValidator, tryCountValidator } from './Validators/racingCarValidator.js';
+
+import elements from './Utils/elements.js';
+import { alertMessage } from './Utils/utils.js';
+import { $$ } from './Utils/dom.js';
+
+import { CELEBRATE_MESSAGE } from './constants.js';
+
+export default class CarGameManager {
+  constructor() {
+    this.carGameView = new CarGameView();
+    this.bindEvents();
+  }
+
+  initGame() {
+    elements.$inputNameButton.disabled = false;
+    elements.$inputTryCountButton.disabled = false;
+
+    this.carGameView.init();
+    this.errorMessage = '';
+  }
+
+  bindEvents() {
+    this.bindInputCarNamesEvent();
+    this.bindInputTryCountEvent();
+    this.bindResetEvent();
+  }
+
+  bindInputCarNamesEvent() {
+    elements.$inputNameForm.addEventListener('submit', this.carNamesInputHandler.bind(this));
+  }
+
+  bindInputTryCountEvent() {
+    elements.$inputTryCountForm.addEventListener('submit', this.tryCountInputHandler.bind(this));
+  }
+
+  bindResetEvent() {
+    elements.$resetBtn.addEventListener('click', () => {
+      this.initGame();
+    });
```

Comment:
이렇게 작성할수도 있을 것 같아요 😄 
```suggestion
 elements.$resetBtn.addEventListener('click', this.initGame);
```